direct/transport/open_other.go

```diff
@@ -22,7 +22,7 @@ func OpenTPM(path ...string) (TPM, error) {
 	if err != nil {
 		return nil, err
 	}
-	return &LocalTPM{
```

Comment:
Here (and in open_windows.go) we need the `OpenTPM` function to return `TPMCloser`.

Reply:
Fixed.